CHANGES

```diff
@@ -16,6 +16,11 @@
      (CVE-2019-1552)
      [Richard Levitte]
 
+  *) Compute ECC cofactors if not provided during EC_GROUP construction. Before
+     this change, EC_GROUP_set_generator would accept order and/or cofactor as
+     NULL. After this change, only the cofactor parameter can be NULL. It also
+     does some minimal sanity checks on the passed order.
+     [Billy Bob Brumley]
```

Comment:
nit: The CHANGES entry should go to the top of the top section.

Reply:
Fixed in 2f808454c6aee59e78365cc42c3135b6fae0c356